### PR TITLE
Print error message when server fails to initialize

### PR DESCRIPTION
**Pull request by tkonolige**

### CODE REVIEW

File: autoload/lsp.vim [hunk 1]
```diff
@@ -201,6 +201,12 @@ function! s:throw_step_error(s) abort
     call a:s.callback(a:s.result[0])
 endfunction
 
+function! s:error_msg(msg) abort
```
Comment (prabirshrestha) [inline autoload/lsp.vim]: can we move this to utils?

Comment (tkonolige) [reply autoload/lsp.vim]: Done

File: autoload/lsp/utils.vim [hunk 2]
```diff
@@ -88,3 +88,9 @@ endif
 function! lsp#utils#empty_complete(...) abort
     return []
 endfunction
+
+function! s:error_msg(msg) abort
```
Comment (prabirshrestha) [inline autoload/lsp/utils.vim]: need to change this to `lsp#utils#error(msg)`

Comment (tkonolige) [reply autoload/lsp/utils.vim]: Done

**Comment by prabirshrestha** [comment]
Thanks

**Pull request merged**